thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/dashboard/ThirdEyeDashboardApplication.java

```diff
@@ -160,14 +155,12 @@ public void run(ThirdEyeDashboardConfiguration config, Environment env)
 
     AnomalyFunctionFactory anomalyFunctionFactory = new AnomalyFunctionFactory(config.getFunctionConfigPath());
```

Comment:
AnomalyFunctionFactory is not used.

Reply:
`anomalyFunctionFactory` is used to initialize `AnomalyResource`, `DataResource`, and `AnomaliesResource`. Could we confirm all these three resources are not used?

Reply:
@jihaozh can you check which resource is used? I believe most endpoints in AnomalyResource and AnomaliesResource are not used. From Harley's doc only "/anomalies/search/time/" is used, but let's double check with them.

Reply:
I think `AnomalyResource` is not used.  In `AnomaliesResource`, only the search endpoints listed by Harley is used. `DataResource` is being used, but it shouldn't need `anomalyFunctionFactory` anymore.

Reply:
When we search for anomalies by using `/anomalies/search/time/`, we construct `AnomalyDetails`, and it requires `BaseAnomalyFunction` to do it.